specification/containerservice/resource-manager/Microsoft.ContainerService/preview/2022-04-02-preview/managedClusters.json

```diff
@@ -4868,13 +4871,15 @@
       "type": "string",
       "enum": [
         "Ubuntu",
-        "CBLMariner"
+        "CBLMariner",
+        "Windows2019",
+        "Windows2022"
       ],
       "x-ms-enum": {
         "name": "OSSKU",
         "modelAsString": true
       },
-      "description": "Specifies an OS SKU. This value must not be specified if OSType is Windows."
+      "description": "Ubuntu or CBLMariner for Linux node pool. The default is Ubuntu if OSType=Linux. Windows2019 or Windows2022 for Windows node pool. The default is Windows2019 if OSType=Windows."
```

Comment:
minor: consider changing language slightly to clarify:
"Specifies the OS SKU used by the agent pool. If not specified, the default is Ubuntu if OSType=Linux or Windows2019 if OSType=Windows"

Reply:
Thanks for your guidance. Updated the description.

Reply:
@matthchr @ShiqianTao We may change the default OSSku from Windows2019 to Windows2022 when we decide to deprecate WS2019

Reply:
should we update the description for Windows to `the default is Windows2019 if OSType=Windows and will be changed to Windows2022 after Windows2019 is deprecated`

Reply:
Yes, it's more clear. Updated the description. I split it into two sentences.